command/testdata/providers/modules/main.tf

```diff
@@ -10,6 +10,6 @@ provider "bar" {
   version = "2.0.0"
 }
 
-module "child" {
+module "kiddo" {
   source = "./child"
```

Comment:
Renamed so that the module didn't match the source directory, so we can verify that the name is shown in the tree.

Reply:
love kiddo/kinder, excellent choices